modules/integration/src/test/scala/scala/cli/integration/RunPipedSourcesTestDefinitions.scala

```diff
@@ -202,4 +202,14 @@ trait RunPipedSourcesTestDefinitions { _: RunTestDefinitions =>
       expect(output == expectedOutput)
     }
   }
+
+  test("running script snipped should not create workspace dir in cwd") {
```

Comment:
```suggestion
  test("running a script snippet should not create the workspace dir in cwd") {
```

Reply:
btw. this test should go to either `RunSnippetTestDefinitions` or `DefaultTests`